lxl-web/src/lib/components/Modal.svelte

```diff
@@ -45,8 +59,8 @@
 <dialog
 	class="fixed top-0 left-0 flex h-screen max-h-full w-screen max-w-full"
 	tabindex="-1"
-	on:click|self={handleBackdropClick}
```

Comment:
Seems like the `self` modifier isn't needed since `handleBackdropClick` checks if the click came from the element itself?